inst/plumber/17-arguments/plumber.R

```diff
@@ -0,0 +1,126 @@
+#* Works with query string aruguments
+#* @serializer print
+#* @post /named/<a>/<b>
```

Comment:
Could we name this one `/bad-practice/<a>/<b>`, the other one `good-practice/<a>/<b>`, and have the examples jump right into a failing case for `bad-practice`?